jupyterhub/templates/hub/pdb.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "jupyterhub.labels" . | nindent 4 }}
 spec:
-  minAvailable: 1
+  minAvailable: {{ .Values.hub.pdb.minAvailable | default 1 }}
```

Comment:
We are duplicating the default values again and making one pointless, in this case the one here in the template. Let's stick with having them only in values.yaml.

Same for the user-scheduler and proxy pdb.